src/sparseml/modifiers/quantization/gptq/pytorch.py

```diff
@@ -46,14 +46,36 @@ class GPTQModifierPyTorch(GPTQModifier):
                 - LayerCompressor.revert_layer_wrappers()
 
     | Sample yaml:
-    |   test_stage:
-    |       obcq_modifiers:
-    |           GPTQModifier:
-    |               sequential_update: True
-    |               dampening_frac: 0.001
-    |               targets: __ALL__
-    |               block_size: 128
-    |               quantize: True
+    | test_stage:
+    |    obcq_modifiers:
+    |      GPTQModifier:
+    |          ignore:
+    |            - "LlamaRotaryEmbedding"
+    |            - "LlamaRMSNorm"
+    |            - "SiLUActivation"
+    |            - "MatMulLeftInput_QK"
+    |            - "MatMulRightInput_QK"
+    |            - "MatMulLeftInput_PV"
+    |            - "MatMulRightInput_PV"
+    |            - "MatMulOutput_QK"
+    |            - "MatMulOutput_PV"
+    |            - "lm_head"
+    |            - "Embedding"
```

Comment:
we don't need any of these except the lm_head, since we no longer quantize by default

Reply:
This is just a example recipe, but I made updates!